### PR TITLE
Added color on active link

### LINKED ISSUE #225: Admin nav not showing active page
**Describe the bug**
Admin navbar does not show the active page

**To Reproduce**
1. Go to /admin
2. Go to any page
3. Notice that it does not show which page is selected

**Expected behavior**
Some kind of color change or underline on the active page

### PR DESCRIPTION
**Pull request by TE4-Mattis-Abrahamsson**
## What issue did you implement or fix?
Fixes #225 

## Summary
- Added color change on active nav items

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## How can this implementation be improved?
- Discuss what color theme should be used, for now, I chose the example color

### CODE REVIEW

File: frontend/src/views/AdminView.vue [hunk 1]
```diff
@@ -13,21 +13,26 @@
             dense
             rounded
           >
-            <router-link
-              v-for="item in items"
-              :key="item.title"
-              :to="item.path"
-              class="d-flex router-link"
+            <v-list-item-group
+              active-class="deep-purple--text text--accent-4"
```
Comment (TE4-jacob-lundberg) [inline frontend/src/views/AdminView.vue]: Is deep-purple really the best color to use?

Comment (Te4-Alexander-Nilsson-Melbostad) [reply frontend/src/views/AdminView.vue]: Holding with jacob here jao

Comment (TE4-Mattis-Abrahamsson) [reply frontend/src/views/AdminView.vue]: Was not sure what color to use, I'm open to suggestions

Comment (TE4-jacob-lundberg) [reply frontend/src/views/AdminView.vue]: perhaps grey lighten-3? https://vuetifyjs.com/en/styles/colors

File: frontend/src/views/AdminView.vue [hunk 2]
```diff
@@ -13,21 +13,25 @@
...
-                </v-list-item-content>
-              </v-list-item>
-            </router-link>
+              <router-link
+                v-for="item in items"
+                :key="item.title"
+                :to="item.path"
+                class="d-flex router-link"
+              >
+                <v-list-item link :active="item.active">
```
Comment (TE4-jacob-lundberg) [inline frontend/src/views/AdminView.vue]: Presume this is built in vuetify functionality

**Pull request merged**